lib/index.js

```diff
@@ -174,8 +174,8 @@ internals.typeDeterminer = joiObject => {
 
     // Collate all meta objects because these need to be pushed into the schema options
     let schemaOptions = joiObject.$_terms.metas.reduce(
-      (options, currentOption) =>
-        Object.assign({}, options._mongoose, currentOption._mongoose),
+      (opts, currentOption) =>
+        Object.assign({}, opts._mongoose, currentOption._mongoose),
```

Comment:
Please can we use the spread operator here? We support Node 10+ and I think the spread operator has been around since Node 8, so we should be good from a compatibility perspective.